src/Rings/mpolyquo-localizations.jl

```diff
@@ -2063,28 +2104,30 @@ end
 # because of massive checks for `iszero` due to memory 
 # management.
 function (f::Oscar.MPolyAnyMap{<:MPolyRing, <:MPolyQuoLocRing, <:Nothing})(a::MPolyRingElem)
-  if !has_attribute(f, :lifted_map)
+  g = get_attribute!(f, :lifted_map) do
     S = domain(f)
     W = codomain(f)
     L = localized_ring(W)
-    g = hom(S, L, lift.(f.img_gens), check=false)
-    set_attribute!(f, :lifted_map, g)
-  end
-  g = get_attribute(f, :lifted_map)
-  return codomain(f)(g(a), check=false)
+    hom(S, L, lift.(f.img_gens), check=false)
+  end::Map{typeof(domain(f)), typeof(localized_ring(codomain(f)))}
+  b = g(a)::MPolyLocRingElem
+  return codomain(f)(numerator(b), denominator(b), check=false)
 end
 
+#=
 function (f::Oscar.MPolyAnyMap{<:MPolyRing, <:MPolyQuoLocRing, <:MPolyQuoLocalizedRingHom})(a::MPolyRingElem)
-  if !has_attribute(f, :lifted_map)
+  g = get_atttribute!(f, :lifted_map) do
     S = domain(f)
     W = codomain(f)
     L = localized_ring(W)
     g = hom(S, L, x -> lift(f.coeff_map(x)), lift.(f.img_gens), check=false)
     set_attribute!(f, :lifted_map, g)
-  end
-  g = get_attribute(f, :lifted_map)
-  return codomain(f)(g(a), check=false)
+  end::Map{typeof(domain(f)), typeof(localized_ring(codomain(f)))}
+
+  b = g(a)::MPolyLocRingElem
+  return codomain(f)(numerator(b), denominator(b), check=false)
 end
+=#
```

Comment:
This seemed a bit awkward. If commenting it out does not cause the tests to fail, then we should remove it.